examples/convergeTest/convergence.cpp

```diff
@@ -0,0 +1,471 @@
+//                        MFEM Analytic Convergence Example
```

Comment:
A better place for this file could be the `tests` directory

Reply:
I didn't know about the tests directory - that's a good idea to move the serendipity convergence test there.  Is there already a unit test for convergence order?  In Firedrake they have something like this that they call "regression tests".  I'm pinging @acfisher on this question as well.

Reply:
This will not be a unit test, but more of a "numerical" test. It could be in `test/` or a subdirectory like `/test/numerical`. We have discussed adding such tests in the past and I believe @v-dobrev may already have a few of them ready.

Reply:
Yes I would say drop it in tests/numerical and at some point we will build up the scaffolding for numerical tests.

Reply:
The tests that @tzanio mentioned are in the `tests-convergence-dev` branch: https://github.com/mfem/mfem/compare/tests-convergence-dev.

Reply:
@v-dobrev, @acfisher and @akgillette, which option do you prefer?

1) move the test from here to the test-convergence-dev branch and prepare it for PR
2) move the tests from  the test-convergence-dev branch here and review as part of this PR
3) keep things as they are and deal with the test-convergence-dev branch later

Reply:
Option 3 is simplest. I think it still a good idea to move this to a sub-directory of `tests`.

Reply:
Yeah lets not hold this one up for the the other branch.  We will sort out the tests in the test-convergence-dev branch.

Reply:
Okay.  I'll wait until @stefanozampini and @dylan-copeland  give their thumbs up, then do `git rm` for the changes to the test file and the example files prior to it being approved by the editors (I think since the files are very small it is okay?).

Reply:
Ok, agreed on option 3.